internal/command/run_test.go

```diff
@@ -1030,7 +1002,7 @@ func TestSendMetadata(t *testing.T) {
 				"BUILDKITE_TEST_ENGINE_TEST_CMD":    "bundle exec rspec",
 				"BUILDKITE_STEP_ID":                 "pqr",
 				// ensure that empty env vars is included in the request
-				"BUILDKITE_TEST_ENGINE_SPLIT_BY_EXAMPLE":          "",
+				"BUILDKITE_TEST_ENGINE_SPLIT_BY_EXAMPLE":          "false",
```

Comment:
A small change in behaviour, when parsing env vars from the `cli` the absence of this var - and any other `bool` vars - will be converted to `false` instead of an empty string.